openassessment/xblock/test/test_openassessment.py

```diff
@@ -100,8 +100,8 @@ def test_student_view_workflow_error(self, xblock):
         # Expect that the page renders even if the update fails
         self.assertIn("OpenAssessmentBlock", xblock_fragment.body_html())
 
-    @ddt.data(('utc', 'April 1, 2014 00:00 UTC'),
-              ('America/Los_Angeles', 'March 31, 2014 17:00 PDT'))
+    @ddt.data(('utc', '2014-03-31 20:00'),
```

Comment:
Why do the dates now all render as ``2014-03-31`` instead of ``March 31, 2014``. Is this change going to be visible to the user, and if so is this the way product expects the dates to be rendered? I think the latter is much more user friendly so I'd rather see that as the default.